apps/reference/_supabase_js/generated/supabase-auth-admin-api.mdx

```diff
@@ -0,0 +1,14 @@
+---
+id: supabase-auth-admin-api
+title: 'Supabase Auth Admin Api'
+slug: /supabase-auth-admin-api
+custom_edit_url: https://github.com/supabase/supabase/edit/master/spec/supabase_js_v2_legacy.yml
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+## Notes
+
+- Any method under the `supabase.auth.admin` namespace will require a `service_role` key.
```

Comment:
Style guide thing: We should avoid future tense (e.g., "will") and stick to present tense as it's generally easier to understand.
```suggestion
- Any method under the `supabase.auth.admin` namespace requires a `service_role` key.
```